.github/workflows/generate-tts-samples.yaml

```
@@ -33,7 +34,7 @@ jobs:
           pip install --verbose sherpa_onnx_bin sherpa_onnx_core sherpa_onnx --no-index -f https://k2-fsa.github.io/sherpa/onnx/cpu.html
 
       - name: supertonic
-        if: true
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)